### ISSUE TITLE
Run marginal syntax check on XSLT..?

### ISSUE THREAD

**Issue by sknorr** [openSUSE/suse-xsl#327]
Stuff like https://github.com/openSUSE/suse-xsl/commit/f4ae4a3 should be easy to prevent if we just ran 

```
xmllint --noent --xpath '//*[local-name(.) = "apply-templates"]/*[local-name(.) = "param"]|//*[local-name(.) = "call-template"]/*[local-name(.) = "param"]' */*.xsl 2> /dev/null
```

on every commit, e.g. via Travis.

**Comment by tomschr** [comment]
This is an idea. However, why not going the full route?

Norman Walsh released a RNG file for XSLT 1.0, 2.0, and 3.0 (see https://github.com/ndw/xslt-relax-ng). We could use his files to "validate" our stylesheets with `jing`.

**Comment by tomschr** [comment]
Ok, this seems solved by #342.
